pkg/models/repository_configuration.go

```diff
@@ -11,8 +11,8 @@ import (
 type RepositoryConfiguration struct {
 	Base
 	Name           string         `json:"name" gorm:"default:null"`
-	Versions       pq.StringArray `json:"version" gorm:"type:text[],default:null"`
-	Arch           string         `json:"arch" gorm:"default:''"`
+	Versions       pq.StringArray `json:"version" gorm:"type:text[],default:['any']"`
```

Comment:
since the `"['any']"` field tag isn't working for versions, it might actually be more clear to remove the default from here. Otherwise, someone might expect that this is doing something.